toucan_connectors/mongo/mongo_connector.py

```diff
@@ -162,12 +162,19 @@ def get_status(self) -> ConnectorStatus:
         # Check databases access
         mongo_client_kwargs = self._get_mongo_client_kwargs()
         mongo_client_kwargs['serverSelectionTimeoutMS'] = 500
-        client = pymongo.MongoClient(**mongo_client_kwargs)
         try:
-            client.server_info()
-        except pymongo.errors.ServerSelectionTimeoutError as e:
-            return ConnectorStatus(status=False, details=self._get_details(2, False), error=str(e))
-        except pymongo.errors.OperationFailure as e:
+            client = pymongo.MongoClient(**mongo_client_kwargs)
+            try:
+                client.server_info()
+            except pymongo.errors.ServerSelectionTimeoutError as e:
+                return ConnectorStatus(
+                    status=False, details=self._get_details(2, False), error=str(e)
+                )
+            except pymongo.errors.OperationFailure as e:
+                return ConnectorStatus(
+                    status=False, details=self._get_details(3, False), error=str(e)
+                )
+        except pymongo.errors.ConfigurationError as e:
```

Comment:
could you please unnest the `try` blocks ?

Reply:
done here : https://github.com/ToucanToco/toucan-connectors/pull/634/commits/c78eac60ab45a0e5ff6db630d67ba233c74491ed